go/apps/api/openapi/openapi-generated.yaml

```
@@ -1774,6 +2111,33 @@ components:
                     "$ref": "#/components/schemas/Pagination"
             additionalProperties: false
+            example:
+                meta:
+                    requestId: req_1234567890abcdef
+                data:
+                    - id: role_1234567890abcdef
+                      name: support.readonly
+                      description: Provides read-only access for customer support representatives
+                      createdAt: 1701425400000
+                      permissions:
+                        - id: perm_1234567890abcdef
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 26 but found 24

(indentation)